roles/splunk/tasks/configure_idxc_manager.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Run splunk version command to check currently installed version
+  command: "{{ splunk_home }}/bin/splunk version --answer-yes --auto-ports --no-prompt --accept-license"
+  register: current_version
+  become: true
+  become_user: "{{ splunk_nix_user }}"
+  changed_when: false
+
+- name: Set splunk version number
+  set_fact:
+    splunk_v_num_only: "{{ current_version.stdout | regex_search ('\\d+\\.\\d+\\.\\d+') }}"
+
+# Splunk version < 8.1.X supports mode=master
+# https://docs.splunk.com/Documentation/Splunk/8.0.9/Indexer/Configuremasterwithserverconf
+- name: Configure clustering stanza for cluster manager node on Splunk version < 8.1.X
```

Comment:
Instead of duplicating this task for different versions, use `set_fact` to set appropriate values for a new var `mode_value` based on the version. Then, you can do all the server.conf configuration in one task.